### PR TITLE
F/favorites

### PR DESCRIPTION
**Pull request by fredrondina96**
Completed User stories 11 and 12 plus passing tests

### CODE REVIEW

File: app/controllers/pets_controller.rb [hunk 1]
```diff
@@ -38,6 +38,9 @@ def destroy
     redirect_to "/pets"
   end
 
+  def favorited
+    binding.pry
```
Comment (caachz) [inline app/controllers/pets_controller.rb]: Pry left in. Please remove

File: app/views/favorite/index.html.erb [hunk 2]
```diff
@@ -1,8 +1,11 @@
 <h1>Favorited Pets</h1><br><br>
 
- <% favorites.contents.keys.each do |pet| %>
- <%= link_to "#{Pet.find(pet).name}", "/pets/#{pet}" %>
-  <%= image_tag("#{Pet.find(pet).image}", :size => "260x180") %>
+ <% favorites.contents.each do |pet_id, favorite| %>
+
+  <%if favorite >= 1%>
+    <%= link_to "#{Pet.find(pet_id).name}", "/pets/#{pet_id}" %>
```
Comment (caachz) [inline app/views/favorite/index.html.erb]: A lot of logic in the view file. Can we push this back to the controller?

File: app/views/pets/show.html.erb [hunk 3]
```diff
@@ -7,6 +7,10 @@
 <p>Shelter: <%= link_to "#{@pet.shelter.name}", "/shelters/#{@pet.shelter.id}" %></p><br>
 <p>Status: <%= @pet.adoption_status %></p><br>
 
-<%= link_to 'Favorite', "/favorite/#{params[:id]}", method: :patch %><br>
+  <% if favorites.pet_check(params[:id]) == false%>
```
Comment (caachz) [inline app/views/pets/show.html.erb]: Mike said we shouldn't use params in our views. I know I did for the first part. We can fix now or refactor later.

File: config/routes.rb [hunk 4]
```diff
@@ -19,6 +19,7 @@
   delete '/pets/:id', to: 'pets#destroy'
 
   patch '/favorite/:pet_id', to: 'favorite#update'
+  delete '/favorite/:pet_id', to: 'favorite#delete'
```
Comment (caachz) [inline config/routes.rb]: I think the restful method name is destroy right?

File: app/views/favorite/index.html.erb (hunk 2 above)
Comment (fredrondina96) [reply app/views/favorite/index.html.erb]: I dont know of a way to have link_to's sent from the controller to the view, for now I cant think of a way to do this differently

File: app/views/pets/show.html.erb (hunk 3 above)
Comment (fredrondina96) [reply app/views/pets/show.html.erb]: I think refactoring later is the best idea, lets focus on getting everything working first

File: app/controllers/pets_controller.rb (hunk 1 above)
Comment (fredrondina96) [reply app/controllers/pets_controller.rb]: pry and redundant method removed

File: config/routes.rb (hunk 4 above)
Comment (fredrondina96) [reply config/routes.rb]: Changed to destroy

**Comment by caachz** [comment]
Confirmed over slack that the delete method was changed to destroy and the pry was taken out. The rest we will leave for refactoring.

**Pull request merged**